src/go/accessd/server/server.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 The Magma Authors.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+
+	"github.com/magma/magma/log"
+	"github.com/magma/magma/protos/magma/sctpd"
+	service_sctpd "github.com/magma/magma/service/sctpd"
+)
+
+const (
+	mmeUpstreamTarget  = "unix:///tmp/mme_sctpd_upstream.sock"
+	mmeGrpcDialTimeout = time.Second
+)
+
+func newMmeGrpcConn() (*grpc.ClientConn, error) {
+	ctx, _ := context.WithTimeout(context.Background(), mmeGrpcDialTimeout)
```

Comment:
looking to the future if we use fx one of the nice things will be to use the injected context rather than context.Background so that fx can control the lifecycle

Reply:
yeah, playing with fx is on my to-do list. will take a look at that next after fixing up a few diffs